.github/workflows/node-badge.yml

```diff
@@ -4,10 +4,11 @@ name: "Generate node comparison badge"
 on:
   push:
     paths:
-      - "src/games/definitions.rs"
+      - "crates/lib/src/games/definitions.rs"
       - ".github/workflows/node-badge.yml"
+      - ".github/workflows/scripts/node-badge.mjs"
   schedule:
-    - cron: '0 0 * * 1' # Update once a week in case node-gamedig has changed
+    - cron: '34 3 * * 2' # Update once a week in case node-gamedig has changed
```

Comment:
Good to merge, suppose this got left here for testing?

Reply:
Good to merge: I changed to random time as most people who set up crons will likely set to `0 0 * * 1`, so severs will be less loaded at random time and we might save some action use time.